_data-prepper/common-use-cases/trace-analytics.md

```diff
@@ -27,7 +27,7 @@ To monitor trace analytics, you need to set up the following components in your
 
 To monitor trace analytics in Data Prepper, we provide three pipelines: `entry-pipeline`, `raw-trace-pipeline`, and `service-map-pipeline`. The following image provides an overview of how the pipelines work together to monitor trace analytics. 
```

Comment:
Data Prepper provides three pipelines that can be used to monitor trace analytics: